subworkflows/local/align_star.nf

```diff
@@ -0,0 +1,88 @@
+//
+// Alignment with STAR
+//
+
+include { STAR_ALIGN          } from '../../modules/nf-core/star/align/main'
```

Comment:
This can be installed via nf-core subworkflows now

Reply:
Fixed in #77 - replaced local version with nf-core version